### PR TITLE
Add koolaid

### PR DESCRIPTION
**Pull request by xtopolis**
Didn't run locally, but:

Screen Shot 2022-09-05 at 4 27 43 PM

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -111,6 +111,10 @@ export const reactionsByPattern = [
     pattern: /\bquote\b/i,
     reactions: [["airquotes", "airquotes-left"]],
   },
+  {
+    pattern: /\boh+? (yeah{0,}?|no+)\b/i,
```
Comment (pabo) [inline src/config.js]: `*` instead of `{0,}?`

**Review by pabo — CHANGES_REQUESTED** [review]
Love it. One small fix.

File: src/config.js (hunk 1 above)
Comment (xtopolis) [reply src/config.js]:
```suggestion
    pattern: /\boh+ (yeah*|no+)\b/i,
```

Comment (xtopolis) [reply src/config.js]: How about this ^

**Pull request merged**